examples/controlnet/requirements_sdxl.txt

```
@@ -0,0 +1,8 @@
+accelerate>=0.16.0
+torchvision
+transformers>=4.25.1
+ftfy
+tensorboard
```

```suggestion
tensorboard
wandb
```

Since we use wandb in the example?

---

I swear to god, I had committed that suggestion. Let me add.